src/lib/UserSetting/Tools/DateTimeFormat/DateTimeFormatterFactoryInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformUser\UserSetting\Tools\DateTimeFormat;
```

Comment:
I think this namespace could be simplified to: 

```suggestion
namespace EzSystems\EzPlatformUser\UserSetting\DateTimeFormat;
```